src/pages/[...slug].tsx

```diff
@@ -435,22 +461,22 @@ interface IParams extends ParsedUrlQuery {
     slug: string[];
 } // Need this to avoid TS errors.
 
-const removesErroringArticles = (contents: ContentItem[]) => {
-    const removeItems = [
-        'PyMongoArrow: Bridging the Gap Between MongoDB and Your Data Analysis App',
-        'new-time-series-collections',
-        'mongodb-charts-embedding-sdk-react/',
-        'build-movie-search-application',
-    ];
+// const removesErroringArticles = (contents: ContentItem[]) => {
```

Comment:
Did we want to keep this in here?

Reply:
Yeah it may be good to have it in case we have any other content items that give us trouble and we need to quickly disable them.